src/jpgimage.cpp

```diff
@@ -459,6 +460,10 @@ namespace Exiv2 {
                 --search;
             }
             else if ( marker == app2_ && memcmp(buf.pData_ + 2, iccId_,11)==0) {
+                if (size < 2+14) {
+                    rc = 3;
```

Comment:
I am not sure if the value of `rc` is important. I copied this value (`rc = 3`) from a few lines up.

Reply:
The `rc` value is only pretends to be a return value, it is just used in line 558 to print a warning. I'd suggest to assign a value larger than 7 to `rc`, as that is not yet used.

Reply:
Thanks. I have changed it to 8.